debian/pixelated-user-agent.postinst

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+export PATH=/bin:/usr/bin:/sbin:/usr/sbin
+
+systemctl restart pixelated-server
```

Comment:
The only problem that this might cause at the moment is that we might have users that don't use systemd or pixelated-server. That's why killing the process would be less presumptive. But if we're comfortable with these pre-requisites for now (we think that we don't have users who will have this break), then I'm okay with it too.